src/main/java/com/github/pires/obd/reader/acra/ACRAPostSender.java

```diff
@@ -0,0 +1,119 @@
+package com.github.pires.obd.reader.acra;
+
+import android.content.Context;
+import org.acra.ReportField;
+import org.acra.collector.CrashReportData;
+import org.acra.sender.ReportSender;
+import org.acra.sender.ReportSenderException;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.DefaultHttpClient;
+import org.apache.http.message.BasicNameValuePair;
+import org.apache.http.protocol.HTTP;
+
+import java.math.BigInteger;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.*;
+
+
+public class ACRAPostSender implements ReportSender {
+    private final static String BASE_URL = "http://acra-rest1.rhcloud.com/acra.php?email=smarinov@abv.bg";
+    private final static String SHARED_SECRET = "my_shared_secret";
```

Comment:
SHARED_SECRET must be the same like in acra.php
My idea was to use only one acra.php script hosted on openshift and only user email can be configurable. Application can have 1 hardcoded default email.. or better email=null if(email != null) ACRA will send log to this email.